packages/table/test/editableCellTests.tsx

```diff
@@ -26,6 +29,14 @@ describe("<EditableCell>", () => {
         expect(elem.find(".pt-editable-content").text()).to.equal("test-value-5000");
     });
 
+    it("renders loading state", () => {
+        const editableCellHarness = harness.mount(<EditableCell loading={true} value="test-value-5000" />);
+        const editableCellElement = editableCellHarness.element.children[0];
+        expect(editableCellElement.textContent).to.equal("");
+        expect(editableCellElement.classList.contains(Classes.LOADING)).to.be.true;
+        expect(editableCellElement.children[0].classList.contains(Classes.SKELETON)).to.be.true;
```

Comment:
_please_ don't rely on DOM structure in tests. use methods like `querySelector(`.${Classes.SKELETON}`)` when you expect some child to exist.